### PR TITLE
Create generate_crossref_doi_metadata.py

### PR DESCRIPTION
**Pull request by knmnyn**
Placeholder for DOI assignation script to be uploaded.

### CODE REVIEW

**Comment by knmnyn** [comment]
Partially addresses #158

**Comment by knmnyn** [comment]
Should refactor month to num given @mbollmann 's utils.py function.  I will get on that later.

**Comment by mjpost** [comment]
Okay, I looked at this—and the main issue that is going to raise problems is that you used the `xml` module, but all our tools are built on `lxml`. 

    from lxml import etree as ET 

I imagine adapting your script [won't be too much work](https://lxml.de/1.3/compatibility.html). If you do this, you can easily get our title parser and indenter by adding the following:

```
from anthology.formatter import MarkupFormatter
from anthology.tools import indent

formatter = MarkupFormatter()
```

Then, you can get the title text by passing the LXML `title` node to the formatter:

```
title = paper.find('title')
o_title = ET.Element('title')
o_title.text = formatter.as_text(title)
```

and so on.

File: bin/generate_crossref_doi_metadata.py [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+import sys
+import time
+import re
+
+# CONSTANTS
+PUBLISHER_PLACE = "Stroudsburg, PA, USA"
+DOI_PREFIX = "10.18653/v1/"
+DEPOSITOR_NAME = "Min-Yen Kan"
+EMAIL_ADDRESS = "kanmy@comp.nus.edu.sg"
+REGISTRANT = "Association for Computational Linguistics"
```
Comment (mjpost) [inline bin/generate_crossref_doi_metadata.py]: I should have mentioned—can't we use `anthology@aclweb.org` here?

**Pull request merged**